charmcraft/templates/init-go-framework/charmcraft.yaml.j2

```diff
@@ -9,9 +9,9 @@ base: ubuntu@24.04
 
 platforms:
   amd64:
-  arm64:
-  ppc64el:
-  s390x:
+  # arm64:
```

Comment:
Perhaps include a note explaining what should be done including which command to run to check the architecture, e.g., `uname -m` and how to interpret the command output?